routers/object.js

```diff
@@ -452,7 +452,13 @@ router.post('/:objectName/requestGaussianSplatting/', async function (req, res)
     // splat status (commonly referred to as "splattus") is
     // {done: boolean, gaussianSplatRequestId: string|undefined}
     try {
-        const splatStatus = await objectController.requestGaussianSplatting(req.params.objectName);
+        let userJWT = null;
+        const credentials = req.headers.authorization;
+        if (credentials && credentials.includes(' ')) {
+            // The credentials will be in the format "Bearer <JWT>", can be split to extract the JWT
+            userJWT = credentials.split(' ')[1];
```

Comment:
Instead of parsing the JWT out and then re-combining it with 'Bearer: ', we can just pass `credentials` through the pipeline all the way to the fetch request, then simply `Authorization: credentials` there.